composer.json

```diff
@@ -23,15 +23,14 @@
         "ext-json": "*",
         "psr/cache": "^1.0",
         "psr/log": "^1.0",
-        "doctrine/inflector": "~1.3",
         "weew/helpers-array": "^1.3",
         "psr/http-client": "^1.0",
-        "psr/http-factory": "^1.0"
+        "psr/http-factory": "^1.0",
+        "alextartan/guzzle-psr18-adapter": "^1.2"
```

Comment:
Revert that change so people can use guzzle 5 adapters

Reply:
I see, i've reverted it